app/models/book_indexing.rb

```diff
@@ -0,0 +1,59 @@
+class BookIndexing
+  include Dynamoid::Document
+
+  VALID_STATES = [
```

Comment:
These are all the states, so why prefix with "VALID_".  Is there an "invalid" state?  Maybe just call "STATES"